controller/user-controller.js

```diff
@@ -0,0 +1,43 @@
+const userService = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname, skintype} =req.body;
+
+    
+
+    if (!email|| !password || !firstname || !lastname){
+      throw new Error("KEY Error!");
+    }
+
+    await userService.signUp(email, password, firstname, lastname, skintype);
+    return res.status(201).send({message: "SIGNUP_SUCCESS"});
+  } catch (err) {
+  res.status(err.statusCode || 400).json({message: err.message});
```

Comment:
들여쓰기를 통해 `catch(err) { ... }` 구문 내부에서 실행되는 코드임을 명시해주세요
 ```suggestion
      res.status(err.statusCode || 400).json({message: err.message});
```